blog/_posts/2025-02-04-highlights-2024.md

```diff
@@ -0,0 +1,248 @@
+- [Scala 3.6 announcement](https://scala-lang.org/news/3.6.2/)
+
+### Scala 2 maintenance
+
+We remain committed to maintaining Scala 2 and publishing new versions as needed.
+
+Scala 2.12 is now under minimal maintenance.
+It is not EOL yet primarily because sbt 1.x, which requires 2.12, is still in wide use.
+
+Scala 2.13 improvements are mostly in the areas of JDK compatibility, warnings and lints, and aligning with Scala 3, especially under the `-Xsource:3` flag.
+
+Sometime this year, we expect to start making additions to the Scala 2.13 standard library (which is shared with Scala 3).
+The scope of these additions remains to be seen.
+Initially, additions will be minor, focused on performance overrides and on addressing oversights in the collections API.
+
+Further reading:
+
+- [Scala 2 maintenance plans blog post](https://www.scala-lang.org/blog/2024/12/16/scala-2-maintenance.html)
+- Release notes: [2.13.16](https://github.com/scala/scala/releases/tag/v2.13.16), [2.13.15](https://github.com/scala/scala/releases/tag/v2.13.15), [2.13.14](https://github.com/scala/scala/releases/tag/v2.13.14), [2.13.13](https://github.com/scala/scala/releases/tag/v2.13.13)
+
+### Experimental WebAssembly backend in Scala.js
+
+Starting with 1.17.0, Scala.js introduces an experimental WebAssembly (Wasm) backend.
+It is designed to be a drop-in replacement for the usual JavaScript backend, albeit with a few [current limitations](https://www.scala-js.org/doc/project/webassembly.html#language-semantics).
+
+The key benefits of using the Wasm backend are:
+
+- Remarkable run-time performance for use cases with minimal JS interop
+- Compact binary format that keeps artifact sizes small, although we don't support any Wasm-specific code size optimizer yet
+
+Supported engines include Node.js 23, Chrome and Firefox, all using some [experimental flags](https://www.scala-js.org/doc/project/webassembly.html#supported-engines).
+Support for standalone Wasm, that can run outside the JavaScript environment, is a work-in-progress.
+
+To try out the Wasm backend in your Scala.js project, check out the [setup section](https://www.scala-js.org/doc/project/webassembly.html#language-semantics) in the Scala.js documentation.
+
+Further reading:
+
+- [Scala.js 1.17 release notes](https://www.scala-js.org/news/2024/09/28/announcing-scalajs-1.17.0/): Experimental Web-Assembly backend and more.
+- [Scala.js 1.16 release notes](https://www.scala-js.org/news/2024/03/19/announcing-scalajs-1.16.0/): Scala.js-specific minifier and more.
+
+### Multi-threading support in Scala Native
+
+Scala Native 0.5 introduces an implementation of `java.lang.Thread` based on system threads, along with Scala and Java concurrency primitives.
+It provides support for synchronized blocks, `@volatile` annotations, final fields, and includes thread-safe implementations of most classes in `java.util.concurrent`, `scala.concurrent` and `scala.collection.concurrent`.
+
```

Comment:
```suggestion

The support is solid enough to allow the Scala parallel collections library to work on Native without code changes.
```

Reply:
I know it's late to be adding text, but I just think this is amazing and will impress readers.